edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -197,24 +207,29 @@ protected override void Load(ContainerBuilder builder)
             // KubernetesDeploymentProvider
             builder.Register(
                     c => new KubernetesDeploymentMapper(
-                        this.deviceNamespace,
-                        this.edgeDeviceHostName,
-                        this.proxyImage,
-                        this.proxyConfigPath,
-                        this.proxyConfigVolumeName,
-                        this.proxyConfigMapName,
-                        this.proxyTrustBundlePath,
-                        this.proxyTrustBundleVolumeName,
-                        this.proxyTrustBundleConfigMapName,
-                        this.apiVersion,
-                        this.workloadUri,
-                        this.managementUri))
+                            this.deviceNamespace,
+                            this.edgeDeviceHostName,
+                            this.proxyImage,
+                            this.proxyConfigPath,
+                            this.proxyConfigVolumeName,
+                            this.proxyConfigMapName,
+                            this.proxyTrustBundlePath,
+                            this.proxyTrustBundleVolumeName,
+                            this.proxyTrustBundleConfigMapName,
+                            this.persistentVolumeName,
+                            this.storageClassName,
+                            this.apiVersion,
+                            this.workloadUri,
+                            this.managementUri))
                 .As<IKubernetesDeploymentMapper>();
 
             // KubernetesServiceProvider
```

Comment:
Are these provider names in comments outdated? I don't see anything named as such in our codebase or Kubernetes Client API. Please remove this and other provider names in comments below.

Reply:
Yes, they're outdated...  They should match what we're looking for, or be removed.